projects/file-parsing/types-converting/csv-struct.go

```diff
@@ -0,0 +1,43 @@
+package filesConverting
+
+import (
+	"encoding/csv"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+func CsvToStruct(textByte []byte) []Score {
+	result := []Score{}
+	s := Score{}
+
+	r := csv.NewReader(strings.NewReader(string(textByte)))
+	records, err := r.ReadAll()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "can't read: %s", err)
+	}
+	nameIndex := -1
+	highScoreIndex := -1
+	i := 0
+	for nameIndex == -1 || highScoreIndex == -1 || i < len(records[0]) {
+		if records[0][i] == "name" {
+			nameIndex = i
+		} else if records[0][i] == "high score" {
```

Comment:
What would happen if you were given a CSV that _didn't_ have a "high score" column?